packages/popover/src/vaadin-popover.d.ts

```diff
@@ -152,20 +152,20 @@ declare class Popover extends PopoverPositionMixin(PopoverTargetMixin(ThemePrope
   hoverDelay: number;
 
   /**
-   * True if the popover overlay is opened, false otherwise.
+   * True if the popover is visible and available for interaction.
```

Comment:
Adjusted wording based on the [MDN description](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/dialog#open) of `<dialog open>`:

> Indicates that the dialog box is active and is available for interaction.